tests/naive.py

```diff
@@ -532,11 +550,30 @@ def apply_include(D, include):
     D[unrestricted_indices] = tmp_swap[mask]
 
 
-def multi_distance_profile(query_idx, T, m, include=None, discords=False):
+def multi_distance_profile(
+    query_idx, T, m, include=None, discords=False, T_subseq_isconstant=None
+):
     excl_zone = int(np.ceil(m / config.STUMPY_EXCL_ZONE_DENOM))
-    d, n = T.shape
+
+    d = T.shape[0]
+    if T_subseq_isconstant is None or callable(T_subseq_isconstant):
+        T_subseq_isconstant = [T_subseq_isconstant] * d
+
+    T_subseq_isconstant = np.array(
+        [rolling_isconstant(T[i], m, T_subseq_isconstant[i]) for i in range(d)]
+    )
+
     Q = T[:, query_idx : query_idx + m]
-    D = multi_mass(Q, T, m, include, discords)
+    Q_subseq_isconstant = T_subseq_isconstant[:, query_idx].reshape(-1, 1)
```

Comment:
Should we use `expand_dims` or `atleast_2d` here instead of `reshape`?

Reply:
Correct! I will fix this one (and other similar cases) in naive.

Reply:
Please do a quick grep for `reshape(-1, 1)` in the codebase and replace where appropriate

Reply:
I fixed the cases in `.py` scripts (in both `./tests/` and `./stumpy/` directories). 

Also: 
I found a few cases in the `.ipynb` notebooks in `docs/` but did not change them. If needed, please let me know and I will fix those too.